### DISCUSSION TITLE
Tab navigation Combobox losing focus

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#8873] [Q&A]
In WPF build hitting tab while being focused on a combobox loses the focus, an additional Tab click has to be implemented to navigate to the next element. Is this a known problem?

**Comment by jeromelaban** [comment]
Thanks for the report. This is not a known issue, but @MartinZikmund probably knows more. Make sure to make a repro sample to show the issue.

**Reply by MartinZikmund** [reply]
Yes, it is quite likely a Popup focus behavior that we don't handle well
